uint8 ID = 0
uint8 BAUD_RATE = 1
uint8 RETURN_DELAY_TIME = 2
uint8 CW_ANGLE_LIMIT = 3
uint8 CCW_ANGLE_LIMIT = 4
uint8 HIGHEST_TEMPERATURE = 5
uint8 LOWEST_VOLTAGE = 6
uint8 HIGHEST_VOLTAGE = 7
uint8 MAX_TORQUE = 8
uint8 STATUS_RETURN_LEVEL = 9
uint8 ALARM_LED = 10
uint8 ALARM_SHUTDOWN = 11
uint8 TORQUE_ENABLE = 12
uint8 LED = 13
uint8 CW_COMPLIANCE_MARGIN = 14
uint8 CCW_COMPLIANCE_MARGIN = 15
uint8 CW_COMPLIANCE_SLOPE = 16
uint8 CCW_COMPLIANCE_SLOPE = 17
uint8 GOAL_POSITION = 18
uint8 MOVING_SPEED = 19
uint8 TORQUE_LIMIT = 20
uint8 LOCK = 21
uint8 PUNCH = 22

uint8 motor_id
uint8 param
int16 value
---
bool success